src/app/sponsor/our-partners/page.js

```diff
@@ -0,0 +1,161 @@
+import PageTitleTemplate from "@/app/components/PageTitleTemplate";
+import Image from "next/image";
+import GDC from "@/app/images/GDGC_UGA.svg";
+import GDCBg from "@/app/images/GDCBackground.png";
+import KTPBg from "@/app/images/KTPBackground.png";
+import ACMBg from "@/app/images/ACMBackground.png";
+import EPIBg from "@/app/images/EpiBackground.png";
+import SOEBg from "@/app/images/SOEBackground.png";
+import UGAHacks from "@/app/images/UGAHacks.png";
+import KTP from "@/app/images/KTP.svg";
+import CSCI from "@/app/images/CSCI.svg";
+import SOE from "@/app/images/SOE.svg";
+import ACM from "@/app/images/ACM.svg";
+import Byte from "@/app/images/ByteX1.svg";
+import Link from "next/link";
+import EmblaCarousel from "@/app/components/EmblaCarousel";
+
+export default function page() {
+  const OPTIONS = { loop: true };
+  const imagesForCarousel = [
+    {
+      image: GDCBg,
+      title: "Google Developer Groups on Campus: UGA",
+      description:
+        "GDGC UGA co-hosts their meetings with DevDogs, getting you in the Google sphere.",
+      link: "https://gdg.community.dev/gdg-on-campus-university-of-georgia-athens-united-states/",
+    },
+    {
+      image: KTPBg,
+      title: "Kappa Theta Pi: Phi Colony",
+      description:
+        "KTP is UGA's first technology frat, developing the next generation of industry professionals",
+      link: "https://ktpgeorgia.com/",
+    },
+    {
+      image: ACMBg,
+      title: "ACM @ UGA",
+      description:
+        "Every week, ACM hosts workshops discussing the latest technologies.",
+      link: "https://acm-uga.github.io/",
+    },
+    {
+      image: EPIBg,
+      title: "Computer Science Career Initiative",
+      description:
+        "CSCI provides resources to assist students in their pursuit of a many technical fields.",
+      link: "https://uga.campuslabs.com/engage/organization/csci",
+    },
+    {
+      image: SOEBg,
+      title: "Society of Entrepreneurs",
+      description:
+        "Learn from and connect with founders and business-savvy individuals at SOE's many events.",
+      link: "https://www.soeuga.com/",
+    },
+    {
+      image: UGAHacks,
+      title: "UGA Hacks",
+      description:
+        "Every spring, UGA Hacks puts on a 3-day, 800+ person hackathon at MLC.",
+      link: "https://ugahacks.com/",
+    },
+  ];
+  return (
+    <div className="w-full section ">
+      <PageTitleTemplate
+        redText="Partners"
+        blackText="Campus "
+        reverse="true"
```

Comment:
I think the space between the title and the next line can be shorter.

Web View:
![image](https://github.com/user-attachments/assets/4e223bae-08a9-4073-ac5d-8e3a59ccbd6b)

Mobile View:
![image](https://github.com/user-attachments/assets/49c3838c-3a38-4faf-a071-950f2bf9f68a)